src/azure-cli/azure/cli/command_modules/servicefabric/tests/latest/recordings/test_update_settings_and_reliability.yaml

```diff
@@ -154,11 +154,10 @@ interactions:
       accept-language:
       - en-US
     method: GET
-    uri: https://graph.windows.net/00000000-0000-0000-0000-000000000000/me?api-version=1.6
+    uri: https://graph.microsoft.com/v1.0/me
```

Comment:
Running this test takes more than 1 hour. Also, it records the identity's PII. Here I manually created the Graph request with a dummy GUID.